web/lib/components/CodyWebChat.tsx

```diff
@@ -205,7 +207,7 @@ export const CodyWebChat: FC<CodyWebChatProps> = props => {
                                         <Chat
                                             chatID={activeChatID}
                                             chatEnabled={true}
-                                            showWelcomeMessage={false}
+                                            showWelcomeMessage={true}
```

Comment:
```suggestion
                                            showWelcomeMessage={false}
```

@vovakulikov we can set it back to false if you think the current welcome message for web won't be helpful

Reply:
Yeah, I think we could do better there, also Welcome message probably should be a bit different from place to place (blob UI side panel and standalone chat page) so I would turn this off for Cody Web for now, I will take a look at this early next week.